f8a_worker/dispatcher/nodes.yml

```diff
@@ -481,6 +481,15 @@
         region_name: 'us-east-1'
         encryption: false
         versioned: true
+  
+    - name: 'S3ManualTagging'
+        import: 'f8a_worker.storages'
```

Comment:
In order to make this work you need to add import of `S3ManualTagging` to `f8a_worker/storages/__init__.py`. This will brake worker start.